tests/common/assert/pipeline-ready.yaml

```diff
@@ -55,12 +55,19 @@ spec:
                   fieldPath: metadata.name
             - name: GOMEMLIMIT
               (value != null): true
+            - name: GOMAXPROCS
+              valueFrom:
+                resourceFieldRef:
+                  containerName: gateway
+                  divisor: "0"
+                  resource: limits.cpu
           name: gateway
           resources:
             requests:
               (memory != null): true
             limits:
               (memory != null): true
+              (cpu != null): true
```

Comment:
since we have it for both limits and requests, can be nice to add it in both places :)